ghcide/src/Development/IDE/Plugin/CodeAction.hs

```diff
@@ -29,30 +49,21 @@ import Development.IDE.Plugin.TypeLenses (suggestSignature)
 import Development.IDE.Types.Exports
 import Development.IDE.Types.Location
 import Development.IDE.Types.Options
-import qualified Data.HashMap.Strict as Map
+import qualified GHC.LanguageExtensions as Lang
+import Ide.PluginUtils (subRange)
+import Ide.Types
 import qualified Language.Haskell.LSP.Core as LSP
-import Language.Haskell.LSP.VFS
 import Language.Haskell.LSP.Types
-import qualified Data.Rope.UTF16 as Rope
-import Data.Char
-import Data.Maybe
-import Data.List.Extra
-import Data.List.NonEmpty (NonEmpty((:|)))
-import qualified Data.List.NonEmpty as NE
-import qualified Data.Text as T
-import Text.Regex.TDFA (mrAfter, (=~), (=~~))
+import Language.Haskell.LSP.Types.Lens (character, end, line, start)
+import Language.Haskell.LSP.VFS
+import OccName (parenSymOcc)
 import Outputable (Outputable, ppr, showSDoc, showSDocUnsafe)
-import Data.Function
-import Control.Arrow ((>>>))
-import Data.Functor
-import Control.Applicative ((<|>))
+import Retrie.GHC (mkVarOcc)
 import Safe (atMay)
-import Bag (isEmptyBag)
-import qualified Data.HashSet as Set
-import Control.Concurrent.Extra (readVar)
-import Development.IDE.GHC.Util (printRdrName)
-import Ide.PluginUtils (subRange)
-import Ide.Types
+import Text.Regex.TDFA (mrAfter, (=~), (=~~))
+import Data.Either (fromRight)
+import Retrie (unpackFS)
+import FieldLabel (flLabel)
```

Comment:
Could you minimise the change to the imports? Otherwise it's guaranteed to generate painful conflicts for other PRs

Reply:
Ah, I applied code-formatter restricted to the import list, but it seems it introduced so much unnecessary change...